Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
+                                <citation type="doi">10.1038/nbt.2198</citation>
+                            </citations>
+                            ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper directory tree
+
+                            ```
+                            graphlan/
+                            ├── graphlan.xml
+                            ├── test_data/
+                                ├── input_tree.txt
+                                ├── png_image.png
+
+                            ```
+
+                            [Details of XML tag sets](https://wiki.galaxyproject.org/Admin/Tools/ToolConfigSyntax)
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/planemo-logo.png" alt="Planemo" class="no-border" height=25/>
+                            #### Building Galaxy Tools
+
+                            <img src="../images/planemo_tool_building_lint.png" alt="Building tools" class="no-border"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            <img src="../images/exercise.png" alt="exercise"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### The Goal
+                            A wrapper for Seqtk, a tool for processing sequence data in FASTA and FASTQ files.
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### Before: install Seqtk and test it
+                            <small>Not need brain</small>
+                            - `planemo conda_init`
+                            - `conda install seqtk`
+                            - `wget https://raw.githubusercontent.com/galaxyproject/galaxy-test-data/master/2.fastq`
+                            - `seqtk seq -A 2.fastq > 2.fasta`
+                            - `cat 2.fasta`
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### Steps
+                            1. Use `planemo tool_init` and fill the template
```

Comment:
I would have add a slide before to check how seqtk is working and a first reflexion on what must be in the wrapper. Maybe with a question to force the reflexion first?